src/sentry/integrations/slack/endpoints/event.py

```diff
@@ -166,7 +166,10 @@ def post(self, request: Request) -> Response:
 
         if slack_request.type == "message":
             data = slack_request.data.get("event")
-            command = data["text"]
+            if data.get("bot_id"):
```

Comment:
In `on_message()` we do:
```python
# if it's a message posted by our bot, we don't want to respond since
# that will cause an infinite loop of messages
if data.get("bot_id"):
    return self.respond()
```
I think you should create a function:
```python
def is_bot(data: Mapping[str, Any]) -> bool:
    """
    If it's a message posted by our bot, we don't want to respond since that 
    will cause an infinite loop of messages. 
    """
    return bool(data.get("bot_id"))
```
So that if we ever need to get more sophisticated in bot detection we can update in one place and both usages can see the note about why we check.